engine/src/main/java/org/terasology/engine/subsystem/common/ConfigurationSubsystem.java

```diff
@@ -46,6 +52,15 @@ public void preInitialise(Context rootContext) {
         config = new Config();
         config.load();
 
+        FlexibleConfigManager flexibleConfigManager = new FlexibleConfigManagerImpl();
+        rootContext.put(FlexibleConfigManager.class, flexibleConfigManager);
+
+        FlexibleConfig renderingFlexibleConfig = new FlexibleConfigImpl();
+        flexibleConfigManager.addFlexibleConfig(new SimpleUri("engine:rendering"), renderingFlexibleConfig);
+
+        flexibleConfigManager.load();
```

Comment:
can't this be part of `config.load()`? We already have an integration to `config.save()`

Reply:
One thing we'll have to be careful about is that FCs get initialized -before- config gets loaded. Not sure if this is possible for every single config.

Not a big deal right now, but just wanted to bring it up.

Reply:
I think the aim is to eventually eliminate Config entirely and leave only the FCmanager to fill those shoes. So, I think it makes sense to keep them in parallel for the time being.